README.md

```diff
@@ -60,40 +60,48 @@ also have committer rights on the repo.
 * [@manishprivet](https://github.com/manishprivet) - Manish Kumar
-* [@marcustisater](https://github.com/marcustisater) - Marcus Tisäter
-* [@mbj36](https://github.com/mbj36) - Mohit kumar Bajoria
 * [@MylesBorins](https://github.com/MylesBorins) - Myles Borins
-* [@ogonzal87](https://github.com/ogonzal87) - Oscar Gonzalez
-* [@ollelauribostrom](https://github.com/ollelauribostrom) - Olle Lauri Boström
 * [@ovflowd](https://github.com/ovflowd) - Claudio Wunder
-* [@pierreneter](https://github.com/pierreneter) - Nguyễn J Huỳnh Long
 * [@rodion-arr](https://github.com/rodion-arr) - Rodion Abdurakhimov
-* [@sagirk](https://github.com/sagirk) - Sagir Khan
-* [@saulonunesdev](https://github.com/saulonunesdev) - Saulo Nunes
-* [@skllcrn](https://github.com/skllcrn) - Christopher
-* [@SMotaal](https://github.com/SMotaal) - Saleh Abdel Motaal
-* [@timothyis](https://github.com/timothyis) - Timothy
-* [@tstreamDOTh](https://github.com/tstreamDOTh) - T Thiyagaraj
-* [@ZYSzys](https://github.com/ZYSzys) - ZYSzys
+
+<details>
+  <summary>Previous Members</summary>
+  
+  These are previous members that were part of the **Nodejs.dev** Team:
+  
+  * [@ahmadawais](https://github.com/ahmadawais) - Ahmad Awais ⚡️
+  * [@alexandrtovmach](https://github.com/alexandrtovmach) - Alexandr Tovmach
+  * [@amiller-gh](https://github.com/amiller-gh) - Adam Miller
+  * [@argyleink](https://github.com/argyleink) - Adam Argyle
+  * [@BeniCheni](https://github.com/BeniCheni) - Benjamin Chen
+  * [@codeekage](https://github.com/codeekage) - Abraham Jr. Agiri
+  * [@designMoreWeb](https://github.com/designMoreWeb) - Divy Tolia
+  * [@imbhargav5](https://github.com/imbhargav5) - Bhargav Ponnapalli
+  * [@jemjam](https://github.com/jemjam) - j'em bones
+  * [@jonchurch](https://github.com/jonchurch) - Jonathan Church
+  * [@kasicka](https://github.com/kasicka) - Zuzana Svetlíková
+  * [@keywordnew](https://github.com/keywordnew) - Manil Chowdhury
+  * [@lancemccluskey](https://github.com/lancemccluskey) - Lance McCluskey
+  * [@LaRuaNa](https://github.com/LaRuaNa) - Onur Laru
+  * [@maddhruv](https://github.com/maddhruv) - Dhruv Jain
+  * [@marcustisater](https://github.com/marcustisater) - Marcus Tisäter
+  * [@mbj36](https://github.com/mbj36) - Mohit kumar Bajoria
+  * [@ogonzal87](https://github.com/ogonzal87) - Oscar Gonzalez
+  * [@ollelauribostrom](https://github.com/ollelauribostrom) - Olle Lauri Boström
+  * [@pierreneter](https://github.com/pierreneter) - Nguyễn J Huỳnh Long
+  * [@sagirk](https://github.com/sagirk) - Sagir Khan
+  * [@saulonunesdev](https://github.com/saulonunesdev) - Saulo Nunes
+  * [@skllcrn](https://github.com/skllcrn) - Christopher
+  * [@SMotaal](https://github.com/SMotaal) - Saleh Abdel Motaal
+  * [@timothyis](https://github.com/timothyis) - Timothy
+  * [@tstreamDOTh](https://github.com/tstreamDOTh) - T Thiyagaraj
+  * [@ZYSzys](https://github.com/ZYSzys) - ZYSzys
+  
+</details>
 
 <!-- ncu-team-sync end -->
```

Comment:
This needs to be at the end of the list, as the ncu tool is usually used to sync the list based on membership

Reply:
I don't think I get your comment. What do you mean with ncu? Node Check Updates?

Reply:
These comments are used by a tool to keep this list in sync https://github.com/nodejs/node-core-utils/blob/main/docs/ncu-team.md